smithy-swift-codegen/src/main/kotlin/software/amazon/smithy/swift/codegen/integration/httpResponse/HttpResponseGenerator.kt

```diff
@@ -36,6 +36,7 @@ class HttpResponseGenerator(
         httpOperations.forEach {
             httpResponseBindingErrorGenerator.render(ctx, it)
             HttpResponseBindingErrorNarrowGenerator(ctx, it, unknownServiceErrorSymbol).render()
+            CodedErrorGenerator(ctx, it, unknownServiceErrorSymbol).render()
```

Comment:
Rendering coded error compliance here does it for every operation.  When this work is integrated with Smithy acceptors, coded error compliance will only be generated for operations that have a waiter with an `errorType` acceptor.

Reply:
Does that mean that the implementation of `CodedErrorGenerator` here will be removed? Assuming it will be implemented somewhere else where it is restricted to only operations that have a waiter with an errorType acceptor?